src/components/ListedSpecies/BehaviorsSection.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect } from 'react';
+import { VStack, Heading, Grid, GridItem, Box, FormControl, FormLabel } from '@chakra-ui/react';
+import { Select } from 'chakra-react-select';
+import { PropTypes } from 'prop-types';
+import { Controller, useFormContext } from 'react-hook-form';
+import CollapsibleSection from '../CollapsibleSection/CollapsibleSection';
+
+const BehaviorsSection = ({
+  behaviorOptions,
+  behaviors,
+  setBehaviors,
+  nestingOptions,
+  nesting,
+  setNesting,
+}) => {
+  const { control, setValue } = useFormContext();
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'setValue' is assigned a value but never used.